dts/common/nordic/nrf54l20.dtsi

```diff
@@ -504,6 +504,7 @@
 				compatible = "nordic,nrf-grtc";
 				reg = <0xe2000 0x1000>;
 				cc-num = <12>;
+				clocks = <&lfxo>;
```

Comment:
since you're adding its `clock`, you can obtain the frequency from it (see e.g. uarte driver, or, can)